docs/source/configuration/overview.mdx

```diff
@@ -316,7 +316,7 @@ See [Tracing in the Apollo Router](./tracing/).
 ### Automatic persisted queries (APQ)
 
 Automatic Persisted Queries (APQ) enable GraphQL clients to send a server the _hash_ of their query string, _instead of_ the query string itself. This can significantly reduce network usage for very large query strings.
-
+****
```

Comment:
is this line intentional ? is this some kinda `<hr>` magic ?

Reply:
hah, it might work.  i've only used `----` to do the same.

Reply:
Fixed in [a51e392](https://github.com/apollographql/router/pull/1774/commits/a51e3925ac55f78f8bcec1f43c0e42766469078a)